### PR TITLE
testing & final adjustments

### PR DESCRIPTION
**Pull request by Ntare22**
add search feature for easy access
add testing and test various features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +68,25 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (bimbolabuari) [inline README.md]: The "Acknowledgements" section should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the[ author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the Creative Commons License.

Kindly add this 🙂

File: README.md [hunk 2]
```diff
@@ -68,3 +68,25 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/d
...
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```
Comment (bimbolabuari) [inline README.md:94]: Kindly deploy the application using Heroku, Netlify, or GitHub Pages, and a link to the deployed project should be present in the README. ✊

File: src/components/Stocks.js [hunk 3]
```diff
@@ -16,26 +18,37 @@ const Stocks = () => {
   const handleCompanyInfo = (symbol) => {
     dispatch(getCompanyInfo(symbol));
   };
+
+  const onSearchChange = (event) => {
+    dispatch(setSearchField(event.target.value));
+  };
+
+  // eslint-disable-next-line max-len
```
Comment (bimbolabuari) [inline src/components/Stocks.js]: Let's remove all disabled Linters, Linting helps with more cleaner and professional code 👍 

Kindly check for other occurrences 🙂

File: src/styles/index.scss [hunk 4]
```diff
@@ -3,3 +3,4 @@
 @import url('./navbar.scss');
 @import url('./cards.scss');
 @import url('./company.scss');
+@import url('./search.scss')
```
Comment (bimbolabuari) [inline src/styles/index.scss:6]: As required, lets stick to the design template given 👍 

> Your design

image

> Original design template 
image

Kindly follow these design guidelines, including:
 - Colors (select one main color for your website).
 - Typography: font face, size and weight.
 - Layout: composition and space between elements.

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @Ntare22, 

You have done a good job so far 🚀  

## Highlights 🎉 

- The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly.
- It connects to the Redux store, using the redux hooks.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Check the comments under the review

- Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
  - a description of the project.
  - a demo of the project features.
  - you should also highlight some interesting piece of code or something you built that you are very proud of.
 
*For recording the video you can use tools like Loom that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.*

-  [ ] Create unit tests for pure functions in your Redux code.
- [ ] Create integration tests for your application using the React Testing Library.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Review by AshakaE — APPROVED** [review]
### Hi @Ntare22  🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**